README.md

```diff
@@ -64,10 +64,10 @@ Diverse Software-Einstellungen kann man über folgende Dateien machen (siehe Kom
 
 Als externe Libraries werden benötigt:
 
- * U8g2 von Oliver, Version 2.26.14
- * Adafruit BME280 Library, Version 1.0.7
- * Adafruit Unified Sensor, Version 1.02
- * IotWebConf, Version 2.3.0
+ * U8g2 von Oliver, Version >=2.26.14
+ * Adafruit BME280 Library, Version >=2.0.0
+ * Adafruit Unified Sensor, Version >=1.0.3
+ * IotWebConf, Version >=2.3.0
```

Comment:
maybe it would make sense to specify:
- minimum requirement
- latest actually tested version